client-course-schedulizer/src/utilities/helpers/caseFunctions.ts

```diff
@@ -172,10 +181,14 @@ export const instructionalMethodCallback = (value: string, { section }: CaseCall
 };
 
 export const sectionCallback = (value: string, params: CaseCallbackParams) => {
-  const sectionParts = value.split("-");
-  prefixCallback(sectionParts[0], params);
-  numberCallback(sectionParts[1], params);
-  letterCallback(sectionParts[2], params);
+  if (value === "--" || !value.trim()) {
+    params.section.isNonTeaching = true;
```

Comment:
Unfortunately this won't set `isNonTeaching` to true in the more common case that there is no `SectionName` column